dist/fyta-plant-card.js

```diff
@@ -236,130 +205,124 @@ const SCHEMA = [
         name: 'state_color_sensor',
         label: 'Show sensor state color',
         selector: { boolean: {} },
+        default: DEFAULT_CONFIG.state_color_sensor,
       },
       {
         name: 'state_color_icon',
         label: 'Show state color icons',
```

Comment:
```suggestion
        label: 'Show state icons color ',
```
As we so not show/hide the icons of the state color but the color for the state icons, the name could be misleading. What do you think?

Reply:
I don't know if that is less confusing. It doesn't sound clearer to me. I think I might have copied this string from another card (might have been the official glance card).

Technically you are exposing the color reflecting the measurement state through by coloring the icon color. But that is way too long. I would keep it as is for the time being.

Reply:
What about "Colored state icons"?

Reply:
As in "Show colored state icons"? That's good but ambiguous as well as the switch toggles from black to colored and not the icon itself.